cg/meta/workflow/microsalt.py

```diff
@@ -164,7 +164,9 @@ def get_samples(self, case_id: str, sample_id: Optional[str] = None) -> List[Sam
         If sample_id is specified, will return a list with only this sample_id.
         Otherwise, returns all samples in given case"""
         if sample_id:
-            return [self.status_db.query(Sample).filter(Sample.internal_id == sample_id).first()]
+            return [
```

Comment:
Would that work here?
```suggestion
            return [self.status_db.session.get_sample_by_internal_id(internal_id=sample_id)
```

Reply:
likely out of scope, just wondering!

Reply:
Session is not required, created a PR for it here https://github.com/Clinical-Genomics/cg/pull/1946